scr/sysbox

```diff
@@ -155,6 +162,7 @@ function sysbox_setup() {
       exit 1
   fi
 
+  setup_sysbox_user
```

Comment:
Can you also add this logic to our sysbox-in-docker initialization script?

Reply:
Done.